instrumentation/opentelemetry-instrumentation-aiohttp-server/pyproject.toml

```diff
@@ -35,11 +35,6 @@ dependencies = [
 instruments = [
```

Comment:
Not related to this pr but dependencies needs to be updated to add opentelemetry-semantic-conventions

Reply:
I don't see any use of `opentelemetry-semantic-conventions` in this instrumentation source code.